samples/segment-destination-example/src/main/AndroidManifest.xml

```diff
@@ -15,7 +15,14 @@
         <activity
             android:name=".main.MainActivity"
             android:exported="false"
-            android:label="@string/title_activity_main" />
+            android:label="@string/title_activity_main">
+            <intent-filter>
+                <data android:scheme="appcues-123" />
```

Comment:
I believe want to update `123` here to `APPCUES_APPLICATION_ID`

Reply:
Whoops, fixed